db/migrate/20160318170635_projects_add_extract_docker_binary.rb

```diff
@@ -0,0 +1,5 @@
+class ProjectsAddExtractDockerBinary < ActiveRecord::Migration
+  def change
+    add_column :projects, :extract_docker_packaged_artifact, :boolean, default: true, null: false
```

Comment:
locally when the migration ran, the existing projects were set to 1 and seems to retain the existing behavior of extracting the tar file.